### DISCUSSION TITLE
How to get it working?

### DISCUSSION THREAD

**Discussion by N1m4rrgon** [Bluemoondragon07/chime-theme#8] [Q&A] [answered]
![grafik](https://user-images.githubusercontent.com/122009141/221417110-0d874d10-ef81-4b0e-a8a0-9df529e059c9.png)

This looks soo good... But how do i actually get this working? I loaded the chime-theme but there is no option to chose the wiki-page style

**Comment by Bluemoondragon07** [comment, marked as answer]
Hi. The wiki-page style is a **css class**, so you have to put it in the frontmatter of the note you want to style. Like this:
```markdown
---
cssClass: wiki-page
---
(Rest of the note)
```
Make sure the frontmatter is at the very top of the note. Otherwise, it may not work.

**Comment by N1m4rrgon** [comment]
Thanks for the fast reply. It works now! :D
